src/housing_package/score.py

```diff
@@ -0,0 +1,193 @@
+# Assignment 2.1
+"""VS Code Auto formatting
+configured vs code to format the entire script
+"""
+import os
+import pickle
+import argparse
+import numpy as np
+import pandas as pd
+import sys
+from sklearn.metrics import mean_absolute_error, mean_squared_error
+
+sys.path.insert(0, os.path.abspath("."))
+from src.housing_package.log_config import LOGGING_DEFAULT_CONFIG, configure_logger
+
+
+def lin_reg_pred(
+    MODEL_PATH: str, housing_prepared: pd.DataFrame, housing_labels: pd.DataFrame
+):
+    """Make model prediction
+    Note: Model should exist in the given path
+
+    Parameters
+    ----------
+
+    param1 : str
```

Comment:
Parameters
------------
MODEL_PATH : str
______path of the model

Reply:
do this for all doc strs

Reply:
done